generators/client/templates/angular/src/test/javascript/spec/app/account/register/_register.component.spec.ts

```diff
@@ -1,12 +1,11 @@
 import { ComponentFixture, TestBed, async, inject, tick, fakeAsync } from '@angular/core/testing';
-import { MockBackend } from '@angular/http/testing';
-import { Http, BaseRequestOptions } from '@angular/http';
 import { Renderer, ElementRef } from '@angular/core';
 import { Observable } from 'rxjs/Rx';
 <%_ if (enableTranslation) { _%>
```

Comment:
Actually it is needed. we are using it below. 
It takes an instance from TestModule's langService though